installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -140,16 +147,23 @@ expects: fedora|centos[/<version>], <ip address>, update, v[1-9].radia.run"
     if [[ $base == $host ]]; then
         host=$host.radia.run
     fi
-    vagrant_dev_private_net=${vagrant_dev_private_net-1}
-    # vbguest is pretty broken so we are defaulting to off for now
-    vagrant_dev_no_vbguest=${vagrant_dev_no_vbguest-1}
+    if [[ ${vagrant_dev_vm_devbox:+1} ]]; then
+        vagrant_dev_no_mounts=${vagrant_dev_no_mounts-1}
+        vagrant_dev_no_nfs_src=${vagrant_dev_no_nfs_src-1}
+        vagrant_dev_no_vbguest=${vagrant_dev_no_vbguest-1}
+        vagrant_dev_private_net=${vagrant_dev_private_net-}
+        vagrant_dev_provision_eth1=${vagrant_dev_provision_eth1-}
+    fi
     if [[ ${vagrant_dev_barebones:+1} ]]; then
         # allow individual overrides
         vagrant_dev_no_dev_env=${vagrant_dev_no_dev_env-1}
         vagrant_dev_no_mounts=${vagrant_dev_no_mounts-1}
         vagrant_dev_no_nfs_src=${vagrant_dev_no_nfs_src-1}
         vagrant_dev_no_vbguest=${vagrant_dev_no_vbguest-1}
     fi
+    vagrant_dev_private_net=${vagrant_dev_private_net-1}
+    # vbguest is pretty broken so we are defaulting to off for now
```

Comment:
This is defaulting to on.

Reply:
There is a "no" in the variable name. So it is setting no vbguest to true which means vbguest as a whole is off.

Reply:
Love those double negatives... It's really hard to get the names right without "no" when they are optional...